hi-ml/Makefile

```diff
@@ -75,3 +75,26 @@ pytest_and_coverage: pip_test call_pytest_and_coverage
 
 # install test requirements and run all tests
 test_all: pip_test flake8 mypy call_pytest_and_coverage
+
+# Smoke tests should all wait for completion and run for 15 minutes maximum.
+# Tests run with the cpath Conda environment because of some odd problems with the hi-ml environment on GPU machines.
+# Some of that can be fixed by pinning torch==1.10.0, but then that triggers problems with distutils.
+define SHARED_ARGS
+--conda_env ../hi-ml-cpath/environment.yml --cluster=pr-gpu --max_run_duration=15m --wait_for_completion
+endef
+
+# HelloWorld model training on a single node, submitted using the v1 SDK
+smoke_helloworld_v1:
+	python src/health_ml/runner.py --model=health_ml.HelloWorld --strictly_aml_v1=True --tag smoke_helloworld_v1 ${SHARED_ARGS}
+
+# HelloWorld model training on a single node, submitted using the v2 SDK
+smoke_helloworld_v2:
+	python src/health_ml/runner.py --model=health_ml.HelloWorld --strictly_aml_v1=False --tag smoke_helloworld_v1 ${SHARED_ARGS}
```

Comment:
I think this tag should be smoke_hello_world_v2

Reply:
good spot! done